wherehows-web/app/templates/application.hbs

```diff
@@ -1,18 +1,22 @@
 {{#if session.isAuthenticated}}
+  {{application/banner-alerts}}
+
   {{partial "navbar"}}
+  {{!-- banner-alert-offset pushes the app view down to show the banner at the top above the navbar --}}
+  <div class="app-container {{if banners.isShowingBanners "banner-alert-offset"}}">
```

Comment:
Does the page need to be offset when a banner is being shown? I think like a modal or a dialog, the intent is to overlay the page, and then the user can dismiss this after acknowledging the banner? or is this an ArtDeco pattern?

Reply:
It's the behavior prescribed by artdeco patterns. It seems weird to me as well, but I think the point is that offsetting the whole page draws definite attention to the alert?